tests/targets/test_android.py

```diff
@@ -146,7 +153,8 @@ def test_check_configuration_tokens(self):
             self.target_android.check_configuration_tokens()
         assert m_check_configuration_tokens.call_args_list == [mock.call([])]
 
-    def test_install_android_sdk(self):
+    @pytest.mark.parametrize("platform", ["linux", "darwin",])
```

Comment:
**nitpick:** In this case, I think that I would remove the trailing comma

Reply:
Addressed